tasks/configure.yml

```diff
@@ -1,5 +1,17 @@
 ---
 
+- name: Add thinpool configuration
+  become: yes
+  template:
+    dest: /etc/docker/daemon.json
+    group: root
+    mode: 0444
+    owner: root
+    src: daemon.json.j2
+  when: docker.thin_pool_device is not none and ansible_distribution_release != "trusty"
```

Comment:
Will thin pool definitely not work on trusty? If so then perhaps we should have an assert that craps out a bit earlier.